build/lib/release.sh

```diff
@@ -193,17 +193,46 @@ function kube::release::package_node_tarballs() {
   done
 }
 
+# Package up all of the server binaries in docker images
+function kube::release::build_server_images() {
+  # Clean out any old images
+  rm -rf "${RELEASE_IMAGES}"
```

Comment:
@spiffxp @BenTheElder @ixdy Only concern is now we have hooked in this new code path with the existing release path, and this command in particular is new. If we have any hiccups with the release build, this command would be my first suspect. I don't foresee any issues, but this is the one difference I see.

Reply:
yeah, FWIW discussed this with ixdy and put the "refactor this new logic to be shared with the existing build" in a seperate commit, so we can revert that commit if necessary and still have the new functionality even. I was pretty wary to touch the existing release logic

Reply:
We also ran `pull-kubernetes-cross` on this PR and the output looks right. If anything does somehow break I'll immediately file a revert for the second commit, I was happy enough without it myself :-)

Reply:
SGTM. If it breaks, we blame Jeff. ;)

Reply:
bbl, taking a vacation until 1.12 is released.

Reply:
Just remember, I was fine merging it as-is... 🚶